cdap-tms/src/main/java/co/cask/cdap/messaging/service/CoreMessagingService.java

```diff
@@ -204,6 +218,32 @@ public void rollback(TopicId topicId, RollbackDetail rollbackDetail) throws Topi
 
   @Override
   protected void startUp() throws Exception {
+    Queue<TopicId> asyncCreationTopics = new LinkedList<>();
+
+    for (String topic : new HashSet<>(cConf.getTrimmedStringCollection(Constants.MessagingSystem.SYSTEM_TOPICS))) {
+      TopicId topicId;
+      try {
+        topicId = NamespaceId.SYSTEM.topic(topic);
+      } catch (IllegalArgumentException e) {
+        // Ignore invalid topic
+        LOG.warn("Ignore creation of invalid topic '{}'.", topic);
```

Comment:
Ah ok , I see we are making the check here. Ignore my previous comment about configuration check of SYSTEM_TOPICS property.